src/store/creator/reducer.ts

```diff
@@ -20,9 +23,18 @@ export const initialState = {
     status: CreatorStatus.init,
 };
 
+function resetStateExceptColor(state: CreatorState): void {
```

Comment:
Conditions within function names are usually a code smell, imo, and this feels a little awkward. Given that we're struggling with this a bit, it feels a bit like `color` should live elsewhere, since the creator state is meant to be temporary. Would moving it to `common` work, since that's also where the current mode lives?

Reply:
Yeah, I also thought about this. will do.